Patch by Robert Scheck <robert@fedoraproject.org> for ptlib <= 2.10.11, which works
around the backward-incompatibility of changed handling of '\#' to '#'. Note that I
didn't get the portable variant for both versions working...

See also: https://lwn.net/Articles/810071/

--- ptlib-2.10.11/make/common.mak		2013-08-15 01:20:27.000000000 +0200
+++ ptlib-2.10.11/make/common.mak.make43	2020-08-16 23:20:03.111199476 +0200
@@ -370,20 +370,20 @@
 # If not specified, find the various version components in the VERSION_FILE
 
   ifndef MAJOR_VERSION
-    MAJOR_VERSION:=$(strip $(subst \#define,, $(subst $(MAJOR_VERSION_DEFINE),,\
+    MAJOR_VERSION:=$(strip $(subst #define,, $(subst $(MAJOR_VERSION_DEFINE),,\
                    $(shell grep "define *$(MAJOR_VERSION_DEFINE) *" $(VERSION_FILE)))))
   endif
   ifndef MINOR_VERSION
-    MINOR_VERSION:=$(strip $(subst \#define,, $(subst $(MINOR_VERSION_DEFINE),,\
+    MINOR_VERSION:=$(strip $(subst #define,, $(subst $(MINOR_VERSION_DEFINE),,\
                    $(shell grep "define *$(MINOR_VERSION_DEFINE)" $(VERSION_FILE)))))
   endif
   ifndef BUILD_TYPE
-    BUILD_TYPE:=$(strip $(subst \#define,,$(subst BUILD_TYPE,,\
+    BUILD_TYPE:=$(strip $(subst #define,,$(subst BUILD_TYPE,,\
                 $(subst AlphaCode,alpha,$(subst BetaCode,beta,$(subst ReleaseCode,.,\
                 $(shell grep "define *BUILD_TYPE" $(VERSION_FILE))))))))
   endif
   ifndef BUILD_NUMBER
-    BUILD_NUMBER:=$(strip $(subst \#define,,$(subst $(BUILD_NUMBER_DEFINE),,\
+    BUILD_NUMBER:=$(strip $(subst #define,,$(subst $(BUILD_NUMBER_DEFINE),,\
                   $(shell grep "define *$(BUILD_NUMBER_DEFINE)" $(VERSION_FILE)))))
   endif
 
